### PR TITLE
switch to pypi as source; update to 1.6

### PR DESCRIPTION
**Pull request by MuellerSeb**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by MuellerSeb** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
 source:
-  url: https://github.com/pyscience-projects/pyevtk/archive/v{{ version }}.tar.gz
-  sha256: 72fe9a4c9b55349a81fa0c424c4f7daaf7bea84814c178b12c38db17e0b385d6
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1f6be7876a3a005c8258861551da4fe7e44ff1a2e7ff2a93d6dc51deedfda5f4
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (xylar) [inline recipe/meta.yaml]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps
```
Although `grayskull` doesn't add it for some reason, I have run into trouble recently where a feedstock without `--no-deps` was bringing in the PyPI versions of dependencies and including them in the conda package.  Better to keep that flag.

**Review by xylar — APPROVED** [review]
@MuellerSeb, this looks great now, thanks! Do you want to add yourself as a maintainer?

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
Just a quick rerender so you become a code owner and I'll merge.

**Comment by xylar** [comment]
@renefritze, shall we remove you here?  I'm happy to keep you on but wanted to respect your wishes if you'd rather be removed.

**Comment by renefritze** [comment]
> @renefritze, shall we remove you here? I'm happy to keep you on but wanted to respect your wishes if you'd rather be removed.

I would appreciate being removed, only to not need to ignore the activity notifications for this repo 😉

**Pull request merged**